core/config/general_config.go

```diff
@@ -782,6 +783,11 @@ func (c *generalConfig) KeeperRegistrySyncUpkeepQueueSize() uint32 {
 	return c.getWithFallback("KeeperRegistrySyncUpkeepQueueSize", ParseUint32).(uint32)
 }
 
+// KeeperCheckUpkeepGasPriceFeatureEnabled enables keepers to include a gas price when running checkUpkeep
+func (c *generalConfig) KeeperCheckUpkeepGasPriceFeatureEnabled() bool {
+	return c.viper.GetBool(EnvVarName("KeeperCheckUpkeepGasPriceFeatureEnabled"))
```

Comment:
Does this fall back to the default? I think you may want `c.getWithFallback`

Reply:
also right :+1: